docs/using_lagoon/docker_images/php-cli-drupal.md

```diff
@@ -0,0 +1,9 @@
+# php-cli-drupal Image
+
+Based on Lagoon `php-cli` image, `cli-drupal` image has all tools for daily maintenance, plus `drush`.
```

Comment:
we are also installing:
- drupal console
- drupal lancher (which will fallback to drush 8 if there is no site installed drush found)

Reply:
Added :)